python/lsst/sims/alertsim/opsim_utils.py

```diff
@@ -87,27 +87,27 @@ def opsim_query_stack10(opsim_path, objid, radius, opsim_night,
 
         return obs_history
 
-def _convert_obs_to_history(obs_list):
+def _convert_obs_to_history(obs_list, obs_gen):
```

Comment:
You need to update the docstring of this method to reflect that `obs_gen` is no an argument.

Reply:
done